addons/resource/models/resource_calendar.py

```diff
@@ -101,10 +101,15 @@ def default_get(self, fields):
     tz_offset = fields.Char(compute='_compute_tz_offset', string='Timezone offset')
     two_weeks_calendar = fields.Boolean(string="Calendar in 2 weeks mode")
     two_weeks_explanation = fields.Char('Explanation', compute="_compute_two_weeks_explanation")
+    flexible_hours = fields.Boolean(string="Flexible Hours",
+                                    help="When enabled, it will allow employees to work flexibly, without relying on the company's working schedule (working hours).")
+    full_time_required_hours = fields.Float(string="Company Full Time", help="Number of hours to work on the company schedule to be considered as fulltime.")
```

Comment:
```suggestion
    full_time_required_hours = fields.Float(string="Company Full Time", help="Number of hours to work, on the company schedule, to be considered as fulltime.")
```

Reply:
Thanks for the review, I agree that we could put commas for readability, but I'd like to mention that the field `full_time_required_hours` here was migrated from hr_payroll module, and therefore the text here was untouched.

The commit is [here](https://github.com/odoo/enterprise/pull/58599/commits/b5f845e3af24cad62f384cf1760b0ffd5e0f0e4d) in the enterprise branch